enterprise/server/test/integration/remote_execution/bazel_rbe/BUILD

```diff
@@ -5,6 +5,7 @@ go_test(
     size = "small",
     srcs = ["bazel_rbe_test.go"],
     shard_count = 15,
+    target_compatible_with = ["@platforms//os:linux"],
```

Comment:
Ditto - we probably don't want to disable this

Reply:
ack